test/specs/scale.time.tests.js

```diff
@@ -617,21 +617,22 @@ describe('Time scale tests', function() {
 		});
 
 		it('should build the correct ticks', function() {
-			// Where 'correct' is a two year spacing.
-			expect(getTicksLabels(this.scale)).toEqual(['2005', '2007', '2009', '2011', '2013', '2015', '2017', '2019']);
+			expect(getTicksLabels(this.scale)).toEqual(['2005', '2006', '2007', '2008', '2009', '2010', '2011', '2012', '2013', '2014', '2015', '2016', '2017', '2018']);
```

Comment:
Do these new labels fit nicely in the chart when rendered? i.e. is there any overlap?

Reply:
Here's a screenshot:

![Screenshot from 2019-10-01 21-14-26](https://user-images.githubusercontent.com/322311/66017685-ae2b2f00-e490-11e9-8369-b68447d2f223.png)